ckan/model/system_info.py

```diff
@@ -53,7 +53,6 @@ def get_system_info(key: str, default: Optional[str]=None) -> Optional[str]:
     ''' get data from system_info table '''
     try:
         obj = meta.Session.query(SystemInfo).filter_by(key=key).first()
-        meta.Session.commit()
```

Comment:
@pdelboca it look like you add this at #6932 
Do you think it is worth keeping it?